### PR TITLE
new: Add Lab 2.15 release notes

### PR DESCRIPTION
**Pull request by tonilastre**
### Description

Lab 2.15 release notes.

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

-

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [x] My changes generate no new warnings or errors
- [x] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

File: pages/release-notes.mdx [hunk 1]
```diff
@@ -122,6 +122,27 @@ Memgraph (v2.11.1 and lower).
...
+
+{<h3> New features and improvements </h3>}
+
+- Share a query with other users with access to the same
+  Lab and Memgraph. Works only on Lab in Docker and with
+  enterprise license.
+- You can configure quick-connect to connect to Memgraph
+  with an encrypted connection with the new environment
+  variable `QUICK_CONNECT_MG_IS_ENCRYPTED`.
+- Size of the Lab Docker image has been decreased by 5x.
```
Comment (kgolubic) [inline pages/release-notes.mdx:136]: Maybe replace with :

The size of the Lab Docker image has been reduced to one fifth of its original size.

**Review by kgolubic — APPROVED** [review]
Everything looks OK. I've left one comment. You can accept it or ignore it.

File: pages/release-notes.mdx (hunk 1 above)
Comment (tonilastre) [reply pages/release-notes.mdx:136]: Yep, makes sense. I will change it.

Comment (tonilastre) [inline pages/release-notes.mdx:136]:
```suggestion
- The size of the Lab Docker image has been reduced to
  one-fifth of its original size.
```

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Pull request merged**